autoload/lsp.vim

```diff
@@ -214,7 +214,11 @@ function! s:call_did_save(buf, server_name, result, cb) abort
 endfunction
 
 function! s:on_text_document_did_close() abort
-    call lsp#log('s:on_text_document_did_close()', bufnr('%'))
+    let l:buf = bufnr('%')
+    call lsp#log('s:on_text_document_did_close()', l:buf)
+    if has_key(s:file_content, l:buf)
```

Comment:
Do we need ref count here so it works if the buffer is opened in splits or tabs?

If ref count is implemented we could also go ahead and implement `textDocument/close`.
https://github.com/prabirshrestha/vim-lsp/issues/26 . Ok with this `textDocument/close` being in new PR but I think we do need ref count in this PR.

Reply:
It should be removed when BufWipeout. Will fix.